frontend/src/pages/AllChatsPage.jsx

```diff
@@ -45,7 +85,21 @@ export default function AllChatsPage() {
     return (
         <div className="AllChatsPage FullHeight LightBackground">
             <ChatInboxHeader />
-            <ChatInbox chats={matches} users={matchedUsers} />
+            <ChatInbox chats={unblockedMatches} users={matchedUsers} />
+
+            <div className={blockedMatches.length === 0 ? 'Hidden' : ''}>
+                <div className="HiddenInbox">
+                    <h5>Hidden Conversations</h5>
+                    <button type="button" className="btn" onClick={handleInboxOpening}>
+                        {!isOpen ? (
+                            <FontAwesomeIcon icon={faChevronDown} />
+                        ) : (
+                            <FontAwesomeIcon icon={faChevronUp} />
+                        )}
```

Comment:
Nicely done adding the chevron!

Another way to write this is to save the icon as a variable and then insert that variable into the layout so that the component JSX/HTML becomes easier to read:

```jsx
const icon = isOpen
  ? (<FontAwesomeIcon icon={faChevronUp} />)
  : (<FontAwesomeIcon icon={faChevronDown} />)
...
return (
    ...
    <h5>Hidden Conversations</h5>
    <button type="button" className="btn" onClick={handleInboxOpening}>
      {icon}
    </button>
    ...
)
```

No need to change it, just wanted to show an example. This is useful when the indentation level and ternaries are getting very deep and you need a way to reduce the number of elements in one block of code so that it can be easier to read.